libraries/botframework-schema/etc/botframework-schema.api.md

```diff
@@ -46,7 +46,7 @@ export interface Activity {
     code?: EndOfConversationCodes | string;
     conversation: ConversationAccount;
     deliveryMode?: DeliveryModes | string;
-    entities?: Entity[];
+    entities?: Entity & {[key: string]: any})[];
```

Comment:
This is incorrect with the change below which indicates that the `Entity` will take KvP mapping string to any. Was this change performed by hand?

Reply:
reverted this change after talking to steven